generated/nidcpower/unit_tests/mock_helper.py

```diff
@@ -231,21 +231,37 @@ def niDCPower_ExportSignal(self, vi, signal, signal_identifier, output_terminal)
     def niDCPower_FetchMultiple(self, vi, channel_name, timeout, count, voltage_measurements, current_measurements, in_compliance, actual_count):  # noqa: N802
         if self._defaults['FetchMultiple']['return'] != 0:
             return self._defaults['FetchMultiple']['return']
+        # voltage_measurements
         if self._defaults['FetchMultiple']['voltageMeasurements'] is None:
             raise MockFunctionCallError("niDCPower_FetchMultiple", param='voltageMeasurements')
         a = self._defaults['FetchMultiple']['voltageMeasurements']
-        for i in range(min(len(voltage_measurements), len(a))):
-            voltage_measurements[i] = a[i]
+        try:
+            voltage_measurements_ref = voltage_measurements.contents
+        except AttributeError:
+            voltage_measurements_ref = voltage_measurements
+        for i in range(min(len(voltage_measurements_ref), len(a))):
```

Comment:
Why pick the minimum of the two? Wouldn't you want an assertion if the sizes are different?

Reply:
They will be different.
Changed to `assert len(voltage_measurements_ref) >= len(test_value)`